src/libraries/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngKeyCreationParameters.cs

```diff
@@ -46,6 +47,7 @@ public CngKeyCreationParameters()
         /// <summary>
         ///     KSP to create the key in
         /// </summary>
+        [MaybeNull]
```

Comment:
Isn't this only supposed to be used on generics?  Either way, I don't think null can ever come out.

Reply:
It can be used for allowing nullable setters in non-nullable properties, but in this case it looks like this is a mistake, we're actually throwing ArgumentNull